src/test/java/homework_6/exercise2/steps/AssertionsSteps.java

```diff
@@ -0,0 +1,85 @@
+package homework_6.exercise2.steps;
+
+import cucumber.api.DataTable;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class AssertionsSteps extends AbstractBaseSteps {
```

Comment:
I assume, that these step could be combined with the steps from the `AssertionsSteps` exercise_1 pacakge